Automattic/jetpack
# WooCommerce Analytics: require package in the Jetpack plugin

Fixes #35631

## Proposed changes:

> [!IMPORTANT]
> Do not merge this PR until #35754, #35756, and #35857 get merged.

Let's require the WooCommerce Analytics package in the Jetpack plugin, and mark the existing codebase in the Jetpack plugin as deprecated.

### Other information:


## Jetpack product discussion

* Epic: #35511

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* This will require using the WooCommerce plugin on your site, and having created some test products.
* Connect the site to your WordPress.com account.
* Go to Jetpack > Settings > Modules and confirm that the WooCommerce Analytics module is enabled.
* Go to the frontend of the site, and confirm that https://stats.wp.com/s-xxxx.js is enabled, xxxx being the year and week number.
* Load Chrome, and install https://github.com/Automattic/tracks-chrome-extension
* Try checking Tracks events generated on Woo pages. You can follow the instructions in #35139 or #34065 for some examples.
